template/src/App.tsx

```diff
@@ -147,6 +166,16 @@ const TabbedApp = () => {
   return (
     <SafeAreaProvider>
       <NavigationContainer
+        linking={{
+          prefixes: ['plaut-ro.github.io/luna', 'localhost'],
```

Comment:
I'm anticipating the github pages CI action here - that will be the prefix when it's done. Obviously localhost needed for it to work locally (it does)